src/map/npc.cpp

```diff
@@ -5294,20 +5294,29 @@ static const char* npc_parse_mob(char* w1, char* w2, char* w3, char* w4, const c
 		}
 		mob.xs = 0;
 	}
+	else if (mob.xs == 0) {
+		// Both 0 and 1 result in fixed x-coordinate
+		mob.xs = 1;
+	}
 
 	if (mob.ys < 0) {
 		if (w1count > 4) {
 			ShowWarning("npc_parse_mob: Negative y-span %hd for mob ID %d (file '%s', line '%d').\n", mob.ys, mob_id, filepath, strline(buffer, start - buffer));
 		}
 		mob.ys = 0;
 	}
+	else if (mob.ys == 0) {
+		// Both 0 and 1 result in fixed y-coordinate
+		mob.ys = 1;
+	}
 
 	if (mob.num > 1 && battle_config.mob_count_rate != 100) {
 		if ((mob.num = mob.num * battle_config.mob_count_rate / 100) < 1)
 			mob.num = 1;
 	}
 
-	if (battle_config.force_random_spawn || (mob.x == 0 && mob.y == 0))
+	if (battle_config.force_random_spawn || (mob.x == 0 && mob.y == 0)
+		|| (mob.xs <= 1 && mob.ys <= 1 && !map_getcell(mob.m, mob.x, mob.y, CELL_CHKREACH)))
```

Comment:
I wouldn't be okay with having 35 warnings every time I start a vanilla rAthena installation.

Reply:
It was not my intention to introduce warnings and you know that.
But okay, we leave away the check and dont further validate the input, because official data is garbage.